remote/bitbucketserver/client.go

```diff
@@ -0,0 +1,61 @@
+package bitbucketserver
+
+import (
+	"net/http"
+	"crypto/tls"
+	log "github.com/Sirupsen/logrus"
+	"io/ioutil"
+	"encoding/pem"
+	"crypto/x509"
+	"github.com/mrjones/oauth"
+)
+
+
+func NewClient(ConsumerKey string, URL string) *oauth.Consumer{
+	privateKeyFileContents, err := ioutil.ReadFile("/private_key.pem")
+	log.Info("Tried to read the key")
```

Comment:
Yeah sadly on bit bucket server when you're adding an incoming auth you have to add the public key and need the private key to do auth calls :/